core/src/main/java/com/novoda/accessibility/ActionsMenuInflater.java

```diff
@@ -0,0 +1,32 @@
+package com.novoda.accessibility;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.support.annotation.MenuRes;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+
+public class ActionsMenuInflater {
+
+    private final MenuInflater menuInflater;
+    private final Resources resources;
+
+    public ActionsMenuInflater(Context context) {
```

Comment:
maybe a static method `ActionsMenuInflator.from(Context)` to match `LayoutInflater`?

Reply:
commit 8eed69cb7476744bade33c805d35cdc38b98df5b (HEAD -> menu-based-actions, origin/menu-based-actions)
Author: Ataul Munim <hello@ataulm.com>
Date:   Tue Apr 3 13:14:54 2018 +0100

    Replace worky ctor with static factory method to match layoutInflater